containers/pre_built_workers/cxx/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM l.gcr.io/google/bazel:latest
+
+RUN mkdir -p /source/code
+WORKDIR /source/code
+
+# Clone source code into directory
+ARG REPOSITORY=grpc/grpc
+ARG GITREF=master
+
+RUN git clone https://github.com/$REPOSITORY.git .
+RUN git submodule update --init
+RUN git checkout $GITREF
+
+# Build workers and spile the executables to another directory
+RUN mkdir /executables
+RUN bazel --output_user_root=/executables build //test/cpp/qps:qps_worker -c opt
+
+# Copy executables to a new image
+FROM python:3
```

Comment:
When using multiple different images for C++ binaries, you need to be careful about binary compatibility.
- you build qps_worker  with bazel in the `l.gcr.io/google/bazel:latest` image
- then you copy it over to `python:3` image.

Depending on what linux distributions (and their versions) are used by these images, the resulting binary might not work on the destination image. I think many times you'll just get lucky and it will work (as it probably does in this case), but it doesn't work in general.

I think when building C++ even in non-prebuilt mode,  the default images for build and run phases are different (bazel is used for build and cxx image is used for runtime) so this is a potential problem that's not only specific to this PR.

Btw, since lots of the languages actually use gRPC C core native library under the hood, the problem is not only specific to C++ (but also could cause trouble for python, ruby, C# etc.).

I'm not saying this is necessarily something that needs to be fixed immediately, but it's something that we should be aware of (and make sure the "build" and "run" images are always binary compatible).

Reply:
We always verify the support of languages by running an example load test in that language, luckily they worked okay so far. I really appreciate your point, and will keep that in mind.  
Also for cxx pre-built, since we do not need to use start_port_server.py anymore, I replaced python image with debian:buster. 
If anything goes wrong in the future, I will start with a debian version and install bazel manually, then use this debian version in the destination image. This way the binary will be built and run with the same linux distributions.